ui/packages/catalog-ui-search/src/main/webapp/react-component/container/localization/localization.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+const properties = require('properties')
+
+type Keywords = {
+  [key: string]: string
+}
+
+export type Context = {
+  keywords: Keywords
+}
+
+export default React.createContext({
+  keywords: properties.keywords,
```

Comment:
❓ Were React i18n libraries considered?

https://www.npmjs.com/search?q=react%20i18n
https://phraseapp.com/blog/posts/react-i18n-best-libraries/

Reply:
https://github.com/yahoo/react-intl
https://github.com/alibaba/react-intl-universal

These were both considered however as I investigated it seemed like these react components were overkill for what I was trying to do

Reply:
Now that we converted those marionette templates to use jsx, it might not be so bad to pull them in and use them.  We could start migrating the `alias` config to use this new method as well (not in this PR, but later).

Reply:
I agree with this approach. I will start making these changes